### PR TITLE
REFACTOR README.md sections into admin manual

### PR DESCRIPTION
**Pull request by fgalan**
Sections related Installing and Running Orion has been moved from master README.md to administration manual, in order they get included in the readthedocs manuals. README.md now include links to the admin manual .md documents in those places.

In addition, this PR fixes the ToC problems described in issue #1136

### CODE REVIEW

File: doc/manuals/admin/database_model.md [hunk 1]
```diff
@@ -86,55 +86,55 @@ Fields:
 Example document:
 
 ```
- {
```
Comment (fgalan) [inline doc/manuals/admin/database_model.md]: JSON hasn't changed actually, but for some reason github shows this as change...

NTC

File: doc/manuals/admin/database_model.md [hunk 2]
```diff
@@ -178,40 +178,40 @@ Fields:
...
```
Comment (fgalan) [inline doc/manuals/admin/database_model.md]: JSON hasn't changed actually, but for some reason github shows this as change...

NTC

File: doc/manuals/admin/database_model.md [hunk 3]
```diff
@@ -254,39 +254,39 @@ Fields:
...
```
Comment (fgalan) [inline doc/manuals/admin/database_model.md]: JSON hasn't changed actually, but for some reason github shows this as change...

NTC

File: doc/manuals/admin/database_model.md [hunk 4]
```diff
@@ -322,29 +322,29 @@ Fields:
...
```
Comment (fgalan) [inline doc/manuals/admin/database_model.md]: JSON hasn't changed actually, but for some reason github shows this as change...

NTC

File: README.md [hunk 5]
```diff
@@ -56,209 +44,13 @@ If this is your first contact with the Orion Context Broker, it is highly recomm
...
-
-Sometimes the above commands fails due to yum cache. In that case, run
-`yum clean all` and try again.
-
-If you are upgrading using the RPM file, then first download the new package from the [FIWARE Files area](https://forge.fiware.org/frs/?group_id=7). Look for the "DATA-OrionContextBroker" entry.
-
-Then upgrade the package using the rpm command (as root):
-
-    rpm -U contextBroker-X.Y.Z-1.x86_64.rpm
+Orion Context broker is described in detail in [the following document](doc/manuals/admin/install.md).
```
Comment (jmcanterafonseca) [inline README.md]: Can we say just 'See the [Admin Manual](doc/manuals/admin/install.md)' .

File: README.md [hunk 6]
```diff
@@ -56,209 +44,13 @@ If this is your first contact with the Orion Context Broker, it is highly recomm
...
-    /etc/init.d/contextBroker status
-
-If broker is running you will get:
-
-    Checking contextBroker...                         Running
-
-If broker is not running you will get:
-
-    Checking contextBroker...                         Service not running
+How to run Orion Context Broker is described in detail in [the following document](doc/manuals/admin/running.md).
```
Comment (jmcanterafonseca) [inline README.md]: Same comment as above

File: README.md (hunk 5 above)
Comment (fgalan) [inline README.md]: What about?

```
Orion Context broker is described in detail in [the corresponding section of the Admin Manual](doc/manuals/admin/install.md).
```

As general rule, I think that the more precise the link to the information that the user wants/needs is, the better.

File: README.md (hunk 6 above)
Comment (fgalan) [inline README.md]: Same response :)

File: README.md (hunk 5 above)
Comment (jmcanterafonseca) [inline README.md]: What about

Build and Install documentation for Orion Context Broker can be found at ...

**Comment by jmcanterafonseca** [comment]
LGTM but I would be happier if the spurious changes detected by Github on the JSON/BSON are fixed ... maybe it is an editor issue ...

File: README.md (hunk 5 above)
Comment (fgalan) [inline README.md]: Fixed in 1271baf

File: README.md (hunk 6 above)
Comment (fgalan) [inline README.md]: Fixed in 1271baf

**Comment by fgalan** [comment]
JSON "diff glitches" has been solved. Please, have a look to the final version of the PR and merge if you find it ok. Thx!

**Comment by jmcanterafonseca** [comment]
LGTM and landing

**Pull request merged**